### PR TITLE
Log bazel mapping differences from analyzer

### PR DESCRIPTION
**Pull request by helin24**
This should not change behavior, as the old analyzer path mapping is the one that will be used for file mapping. I'm only using the new bazel mapping package to see whether the path generated is the same, logging if not.

### CODE REVIEW

**Comment by stevemessick** [comment]
It looks like you need to add a dependency to the Gradle script. `com.android.internal.inputmethod` isn't being found.

**Comment by helin24** [comment]
> It looks like you need to add a dependency to the Gradle script. `com.android.internal.inputmethod` isn't being found.

I'm confused about why this is here - it looks like I added it but I'm not sure why it would be used in this file.

File: flutter-idea/third_party/vmServiceDrivers/org/dartlang/vm/service/VmService.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public class VmService extends VmServiceBase {
   /**
```
Comment (stevemessick) [inline flutter-idea/third_party/vmServiceDrivers/org/dartlang/vm/service/VmService.java:83]: The remaining files were all generated from a script, right? I'm not checking them.

**Review by stevemessick — CHANGES_REQUESTED** [review]
Minor nit in the code, but the Gradle script is broken.

**Review by stevemessick — APPROVED** [review]
That's the easy way to resolve compiler errors :)

File: flutter-idea/third_party/vmServiceDrivers/org/dartlang/vm/service/VmService.java (hunk 1 above)
Comment (helin24) [reply flutter-idea/third_party/vmServiceDrivers/org/dartlang/vm/service/VmService.java:83]: correct

File: flutter-idea/src/io/flutter/run/FlutterPositionMapper.java [hunk 2]
```diff
@@ -321,6 +335,20 @@ protected VirtualFile findLocalFile(@NotNull String uri) {
       if (analyzer != null && !isDartPatchUri(remoteUri)) {
         final String path = analyzer.getAbsolutePath(remoteUri);
         if (path != null) {
+          if (fileFuture != null && path.contains("google3")) {
```
[the remark this answers is not in the document]
Comment (helin24) [reply flutter-idea/src/io/flutter/run/FlutterPositionMapper.java:337]: True, I'll add a workspace check.

**Pull request merged**